app/controllers/alerts_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 class AlertsController < ApplicationController
   before_action :require_signed_in
+  before_action :set_alert, only: [:resolve]
 
   def create
```

Comment:
Now that this AlertsController is doing more than just creating NeedHelpAlerts, it needs a bit of documentation and/or method renaming to clarify that `create` is specifically for creating a `NeedHelpAlert`, while other alert types are created via the update cycle instead of direct action by a user.